### PR TITLE
Added new class names for buttons

### PR DESCRIPTION
**Pull request by sidonaldson**
Can be useful for identifying buttons for styling purposed.
Renamed keyboard-space to keyboard-spaceButton to conform to name
rules, backwards compatible
#22

### CODE REVIEW

File: src/Keyboard.css [hunk 1]
```diff
@@ -51,7 +51,8 @@
 	flex-basis: 56px;
 }
 
-.keyboard-space {
+.keyboard-space,
+.keyboard-spaceButton{
```
Comment (WiaczeslawP) [inline src/Keyboard.css:55]: Let's  use BEM-methodology for naming classes. We have `.keyboard-button` class, and we can add `.keyboard-button--space` class, `.keyboard-button--shift` and so on

File: src/Keyboard.js [hunk 2]
```diff
@@ -194,23 +197,26 @@ export default class Keyboard extends PureComponent {
...
 					<KeyboardButton
 						value={' '}
-						classes="keyboard-space"
+						classes="keyboard-spaceButton"
 						onClick={this.handleLetterButtonClick}
 					/>
 					{inputNode.dataset.type === 'email' ?
 						<KeyboardButton
 							value={'.'}
+							classes={'keyboard-fullstopButton'}
```
Comment (WiaczeslawP) [inline src/Keyboard.js]: I haven't set eslint rules in this project, but it doesn't mean, that we shouldn't keep the code tidy. You pass the `classes`-prop alternately by double quotes and curly braces. Can you please use the only way to do it?

File: src/Keyboard.css [hunk 3]
```diff
@@ -51,7 +51,8 @@
 	flex-basis: 56px;
 }
 
-.keyboard-space {
+.keyboard-space,
```
Comment (WiaczeslawP) [inline src/Keyboard.css:54]: Is it really necessary to keep this class?

File: src/Keyboard.css (hunk 1 above)
Comment (sidonaldson) [reply src/Keyboard.css:55]: I agree BEM would be ideal, but that would mean breaking changes for anyone who used the project and hooked into the CSS classes.

File: src/Keyboard.css (hunk 3 above)
Comment (sidonaldson) [reply src/Keyboard.css:54]: Yes, only to keep it a minor update.

File: src/Keyboard.js (hunk 2 above)
Comment (sidonaldson) [reply src/Keyboard.js]: 👍

**Comment by WiaczeslawP** [comment]
I think it would be also good to set some class for all non-letter buttons for more comfortable manipulations with styling them

**Pull request merged**